### ISSUE TITLE
Randomly dies after loading in to custom systems

### ISSUE THREAD

**Issue by GameWyrm** [Outer-Wilds-New-Horizons/new-horizons#633]
### What Happened?

I have a system that is set as the default system. I have the player and the ship spawn in the same position on a planet. Usually, I just immediately get pulled into the ship. However, rarely when NH loads configs, this seems to mess with collision detection a bit and the game thinks I rammed into the ship, immediately killing me and causing some damage to my ship. After waking up again, I'll always be fine.

### What was supposed to happen?

I was supposed to not die.

### Platform

Steam

### Mods

```Markdown
Vanilla Fix, Menu Framework, Common Camera Utility, Custom Ship Log Modes, Cheat and Debug Menu, Unity Explorer, my WIP mod
```

**Comment by xen-42** [comment]
skill issue

**Comment by JohnCorby** [comment]
fixed by #638???

**Comment by xen-42** [comment]
Considering this fixed by https://github.com/Outer-Wilds-New-Horizons/new-horizons/pull/638
